tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -33,7 +33,7 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc get pod ovsdbserver-nb-0 -o jsonpath='{.metadata.annotations.k8s\.v1\.cni\.cncf\.io/networks-status}' | \
+    oc get pod ovsdbserver-nb-0 -o jsonpath='{.metadata.annotations.k8s\.v1\.cni\.cncf\.io/network-status}' | \
```

Comment:
test looks ok with that

https://logserver.rdoproject.org/76/76/d5cd43c20cfb7ce8704cbf04ba8c1f5177f5582e/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/9eb6d7d/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_minimal_out_2023-08-02T16:21:21EDT.log

```
TASK [ovn_adoption : get podified OVN NB ovsdb-server IP] **********************
changed: [localhost] => {"changed": true, "cmd": "set -euxo pipefail\n\n\noc get pod ovsdbserver-nb-0 -o jsonpath='{.metadata.annotations.k8s\\.v1\\.cni\\.cncf\\.io/network-status}' | jq 'map(. | select(.name==\"openstack/internalapi\"))[0].ips[0]' | tr -d '\"'\n", "delta": "0:00:02.316709", "end": "2023-08-02 16:26:18.199080", "msg": "", "rc": 0, "start": "2023-08-02 16:26:15.882371", "stderr": "+ oc get pod ovsdbserver-nb-0 -o 'jsonpath={.metadata.annotations.k8s\\.v1\\.cni\\.cncf\\.io/network-status}'\n+ jq 'map(. | select(.name==\"openstack/internalapi\"))[0].ips[0]'\n+ tr -d '\"'", "stderr_lines": ["+ oc get pod ovsdbserver-nb-0 -o 'jsonpath={.metadata.annotations.k8s\\.v1\\.cni\\.cncf\\.io/network-status}'", "+ jq 'map(. | select(.name==\"openstack/internalapi\"))[0].ips[0]'", "+ tr -d '\"'"], "stdout": "172.17.0.30", "stdout_lines": ["172.17.0.30"]}
```

Reply:
@booxter looks like this needs a rebase can you sort that out and we can merge today?

Reply:
removed my approval for the rebase

Reply:
I rebased though `looks like this needs a rebase` I wonder how you figured it out while GitHub interface was able to auto-rebase - so I assume there were no conflicts?.. Why do we need to rebase then?

Reply:
see below @booxter we have branch protections enabled which tell us when the branch needs a rebase 
currently "This branch has no conflicts with the base branch".

as a reviewer i cannot know if the rebase would need conflict resolution or not unless i actually try the rebase, hence I asked the author to do it.